src/Service/CloudFront/src/CloudFrontClient.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace AsyncAws\CloudFront;
+
+use AsyncAws\CloudFront\Input\CreateInvalidationRequest;
+use AsyncAws\CloudFront\Result\CreateInvalidationResult;
+use AsyncAws\Core\AbstractApi;
+use AsyncAws\Core\Exception\UnsupportedRegion;
+use AsyncAws\Core\RequestContext;
+
+class CloudFrontClient extends AbstractApi
+{
+    /**
+     * Create a new invalidation.
+     *
+     * @see https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-cloudfront-2019-03-26.html#createinvalidation2019_03_26
+     *
+     * @param array{
+     *   DistributionId: string,
+     *   InvalidationBatch: \AsyncAws\CloudFront\ValueObject\InvalidationBatch|array,
+     *   @region?: string,
+     * }|CreateInvalidationRequest $input
+     */
+    public function createInvalidation2019_03_26($input): CreateInvalidationResult
```

Comment:
Hm.. Should we improve on this name?

Reply:
Hm. Maybe not. This is the name they decided.

Reply:
But they are ugly... Hm... 

I need some input =)

Reply:
Let's discuss this in next PR.

Reply:
Looking at `api2.json`, the model is:

```
    "CreateInvalidation":{
      "name":"CreateInvalidation2019_03_26",
      "http":{
        "method":"POST",
        "requestUri":"/2019-03-26/distribution/{DistributionId}/invalidation",
        "responseCode":201
      },
      "input":{"shape":"CreateInvalidationRequest"},
      "output":{"shape":"CreateInvalidationResult"},
```

meaning that the operation is "keyed" `CreateInvalidation` but for unknown reason it name is `CreateInvalidation2019_03_26`.

The official AwsSdk uses `CreateInvalidation`. And I trully think we should do the same.
by ignoring the `"name"` attribute.

Reply:
I fixed this in https://github.com/async-aws/aws/pull/675

I think "name" is the name of the operation. I use the "key" as name of the method.